test/erc721m/ERC721CMInitializableTest.t.sol

```diff
@@ -30,6 +30,7 @@ contract ERC721CMInitializableTest is Test {
     uint256 public constant GLOBAL_WALLET_LIMIT = 0;
     address public clone;
     uint256 public startTime;
+    uint256 public mintFee = 10000000000000; // 0.00001 ether;
```

Comment:
discussion nit: I always mess up my zeros. If you prefer, you can use 
`uint256 public mintFee = 0.00001 ether;`

It's saved me on precision more than once. Prior to 0.7 you could have done `1 szabo` but they removed finney's and szabos